include/MidiClip.h

```diff
@@ -61,7 +61,7 @@ class LMMS_EXPORT MidiClip : public Clip
 	void updateLength();
 
 	// note management
-	Note * addNote( const Note & _new_note, const bool _quant_pos = true );
+	Note * addNote(const Note& _new_note);
```

Comment:
Ideally remove `_` prefix, too.
```suggestion
	Note* addNote(const Note& _new_note);
```